lib/circuitry/publisher.rb

```diff
@@ -52,7 +51,7 @@ def publish_internal(topic_name, message)
         Timeout.timeout(timeout) do
           logger.info("Publishing message to #{topic_name}")
 
-          topic = TopicCreator.find_or_create(topic_name)
+          topic = Topic.find(topic_name)
```

Comment:
Topic#find and TopicCreator#find_or_create end up calling the same sdk method (create_topic). I don't think we really need Topic.find but I left the interface incase we wanted to add non-create  find flexibility later